### PR TITLE
Disable renovating node version

### PR DESCRIPTION
**Pull request by calebeby**
Tries to disable PR's like this: https://github.com/cloudfour/cloudfour.com-patterns/pull/690

I think this is the config we want: https://github.com/renovatebot/config-help/issues/81#issuecomment-414104278

### CODE REVIEW

File: renovate.json [hunk 1]
```diff
@@ -2,8 +2,10 @@
...
   "rangeStrategy": "pin",
-  "node": {
-    "supportPolicy": ["active"],
-    "rangeStrategy": "bump"
-  }
+  "packageRules": [
+    {
+      "packageNames": ["node"],
+      "enabled": false
+    }
```
Comment (Paul-Hebert) [inline renovate.json:9]: So this is saying:

1. Find any package named "node"
2. Apply the following special rules to it:
3. The only special rule is to entirely disable renovate for it.

Is that right?

Comment (calebeby) [reply renovate.json:9]: Yes that is my understanding

**Comment by calebeby** [comment]
No we don't have a way to test this until we merge it.

Here was my last shot at this: https://github.com/cloudfour/cloudfour.com-patterns/pull/689

_narrator: it did not work_

**Comment by Paul-Hebert** [comment]
Cool, well the code looks good to me based on the linked issue and Renovate docs ¯\\\_(ツ)_/¯

I assume after we merge we just wanna keep an eye on Renovate to make sure it keeps acting correctly?

**Comment by calebeby** [comment]
Yeah, should I merge or should I wait for @spaceninja to review?

**Comment by calebeby** [comment]
After this is merged, I am expecting/hoping that this will auto-close: https://github.com/cloudfour/cloudfour.com-patterns/pull/690

**Comment by Paul-Hebert** [comment]
Your call. I'm tempted to wait for @spaceninja 's review, but worst case this breaks renovate and not our actual codebase, so if you'd prefer to merge now that's fine with me

**Pull request merged**